srcpkgs/gopass-jsonapi/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'gopass-jsonapi'
+pkgname=gopass-jsonapi
+version=1.14.3
+revision=1
+build_style=go
+makedepends="gnupg2"
```

Comment:
`s/gnupg2/gnupg/`

Reply:
:+1: 
I copied it from `gopass` tho, should we change it there aswell?

Reply:
Yes, when updating `gopass`.